### PR TITLE
Added importlib, changed favicon, fixed __version__

### PR DESCRIPTION
**Pull request by zakhar-kogan**
So the changes are:

- #118: As mentioned, ditched `pkg_resources`, now using `importlib_metadata`
- #142: iOS [bullet train emoji](https://emojipedia.org/bullet-train) as favicon (png)
- #158: Fixed the `litellm.__version__` so it's read from `pyproject.toml` (updating in two places would be abysmal and hard for ADHD...)

### CODE REVIEW

File: docs/my-website/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "sharp": "^0.32.5",
```
Comment (ishaan-jaff) [inline docs/my-website/package.json:25]: what does this do

File: litellm/utils.py [hunk 2]
```diff
@@ -1222,7 +1222,7 @@ def litellm_telemetry(data):
         payload = {
             "uuid": uuid_value,
             "data": data,
-            "version": pkg_resources.get_distribution("litellm").version,
+            "version:": importlib.metadata.version("litellm"),
```
Comment (ishaan-jaff) [inline litellm/utils.py:1225]: can we confirm this works ?

File: litellm/utils.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import tiktoken
 
 encoding = tiktoken.get_encoding("cl100k_base")
-import pkg_resources
+import importlib.metadata
```
Comment (ishaan-jaff) [inline litellm/utils.py:10]: if we use import lib do we need to add it as a dependency in pyptoject ?

File: docs/my-website/package.json (hunk 1 above)
Comment (zakhar-kogan) [reply docs/my-website/package.json:25]: Hm, wasn't building (docs) without it 🤔 Can remove and try again

File: litellm/utils.py (hunk 2 above)
Comment (zakhar-kogan) [reply litellm/utils.py:1225]: Yeah, I'll do a video of building -> importing a fresh build -> calling

File: litellm/utils.py (hunk 3 above)
Comment (zakhar-kogan) [reply litellm/utils.py:10]: I don't think so, it's in stdlib:
https://docs.python.org/3/library/importlib.html

File: litellm/utils.py (hunk 2 above)
Comment (zakhar-kogan) [reply litellm/utils.py:1225]: The video with full cycle showing it works for me: https://app.dadan.io/video/share/y4wOwHDV0RwNP26f

**Comment by ishaan-jaff** [comment]
Awesome !

**Comment by ishaan-jaff** [comment]
@zakhar-kogan saw you had nouseresearch and some other custom models on the video ? Curious are you using your custom models with litellm ? 

Can we add support for them better ?

**Pull request merged**